packages/platforms/quanta/x86-64/x86-64-quanta-ix2-rangeley/platform-config/r0/src/python/x86_64_quanta_ix2_rangeley_r0/__init__.py

```diff
@@ -0,0 +1,20 @@
+from onl.platform.base import *
+from onl.platform.quanta import *
+
+class OnlPlatform_x86_64_quanta_ix2_rangeley_r0(OnlPlatformQuanta,
+                                                OnlPlatformPortConfig_48x25_8x100):
+    PLATFORM='x86-64-quanta-ix2-rangeley-r0'
+    MODEL="IX2"
+    SYS_OBJECT_ID=".8.1"
```

Comment:
@jonathantsai-qci The SYS_OBJECT_IDs for the following platforms are all the same:
lb9 - .8.1
ly4r - .8.1
ly9 - .8.1. 

And now this one. These certainly need to be cleaned up. 

Can you please review the SYS_OBJECT_ID assignments for all Quanta platforms and provide unique values moving forward? I'll add a build step which collects all platform information automatically for each build so it can be tracked and reported easily.